.evergreen/run-kerberos-tests.sh

```diff
@@ -1,24 +1,30 @@
 #!/bin/bash
 
-set -o errexit  # Exit the script with error if any of the commands fail
+set -o errexit # Exit the script with error if any of the commands fail
 
 source $DRIVERS_TOOLS/.evergreen/init-node-and-npm-env.sh
 
+bash $DRIVERS_TOOLS/.evergreen/secrets_handling/setup-secrets.sh drivers/enterprise_auth
+source secrets-export.sh
+
 # set up keytab
 mkdir -p "$(pwd)/.evergreen"
 export KRB5_CONFIG="$(pwd)/.evergreen/krb5.conf.empty"
 echo "Writing keytab"
 # DON'T PRINT KEYTAB TO STDOUT
 set +o verbose
 if [[ "$OSTYPE" == "darwin"* ]]; then
-    echo ${KRB5_NEW_KEYTAB} | base64 -D > "$(pwd)/.evergreen/drivers.keytab"
+    echo ${KAYTAB_BASE64_AES} | base64 -D >"$(pwd)/.evergreen/drivers.keytab"
```

Comment:
I fixed my typo in the secrets manager.
```suggestion
    echo ${KEYTAB_BASE64_AES} | base64 -D >"$(pwd)/.evergreen/drivers.keytab"
```